styles.css

```diff
@@ -0,0 +1,93 @@
+body {
```

Comment:
Could you use factors of 16 as your font size to enhance readability, visuals and professionalism?